### ISSUE TITLE
osx arm64 packages contain x86_64 binaries

### ISSUE THREAD

**Issue by manics** [conda-forge/zeroc-ice-feedstock#23]
### Solution to issue cannot be found in the documentation.

- [X] I checked the documentation.

### Issue

See reports
- 3.7.7: https://github.com/conda-forge/zeroc-ice-feedstock/pull/20#issuecomment-1087954281
- 3.6.5 https://github.com/conda-forge/zeroc-ice-feedstock/pull/21#issuecomment-1087946339

Download and extract (`tar -jxvf`) https://anaconda.org/conda-forge/zeroc-ice/3.7.7/download/osx-arm64/zeroc-ice-3.7.7-py39hee96363_0.tar.bz2
```
$ file lib/python3.9/site-packages/IcePy.cpython-39-darwin.so 
lib/python3.9/site-packages/IcePy.cpython-39-darwin.so: Mach-O 64-bit x86_64 bundle, flags:<NOUNDEFS|DYLDLINK|TWOLEVEL|WEAK_DEFINES|BINDS_TO_WEAK>
```

Maybe `setup.py` is using a hardcoded x86_64?
https://github.com/zeroc-ice/ice-packaging/blob/330fc66a497e65e66217741cbbcf481bdf69e91e/ice/pypi/setup.py#L59-L60

### Installed packages

```shell
N/A
```

### Environment info

```shell
N/A
```

**Comment by manics** [comment]
https://conda-forge.org/docs/maintainer/conda_forge_yml.html#azure
```yml
azure:
  # toggle for storing the conda build_artifacts directory (including the
  # built packages) as an Azure pipeline artifact that can be downloaded
  store_build_artifacts: True
```
should allow PR artifacts to be downloaded

Upstream does support arm64, at least for 3.7.7 https://github.com/zeroc-ice/ice/pull/1235

**Comment by psobolewskiPhD** [comment]
I think I posted in the wrong place. 
https://github.com/conda-forge/zeroc-ice-feedstock/pull/25#issuecomment-1091233373
Anyhow, with #25 merged, I made a fresh 3.9 arm64 conda env and was able to install zeroc-ice. The .so file is arm64 and IcePy is importable.
